pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -199,7 +202,7 @@ func (c *Cluster) startMons() error {
 		}
 	}
 
-	logger.Debugf("mon endpoints used are: %s", FlattenMonEndpoints(c.clusterInfo.Monitors))
+	logger.Info("mon endpoints used are: %s", FlattenMonEndpoints(c.clusterInfo.Monitors))
```

Comment:
Is `Info` needed here or is debug ok again? If we keep info level, you'll need to use `Infof()` to get proper formatting

Reply:
Oops leftover.